apps/perf-test/tasks/perf-test.js

```diff
@@ -64,6 +96,19 @@ const iterationsDefault = 5000;
 //  --log-timer-events
 //  --log-source-code
 
+// Analysis
+//  - Why is BaseComponent warnMutuallyExclusive appearing in flamegraphs?
```

Comment:
More reasons to move away from BaseComponent!

Reply:
Totally agree.